README.md

```diff
@@ -215,6 +215,25 @@ $ sudo apt update ; sudo apt -y upgrade
 $ sudo apt -y install wget gnupg2 gnupg-agent dirmngr cryptsetup scdaemon pcscd secure-delete hopenpgp-tools yubikey-personalization
 ```
 
+**Note**
+As of 2023 June, the `hopenpgp-tools` is not part of the latest Debian 12 stable package repositories.
+
+To install it, go to [https://packages.debian.org/sid/hopenpgp-tools](https://packages.debian.org/sid/hopenpgp-tools) to select your architecture and then an ftp server.
+
+Edit `/etc/apt/sources.list` and add the ftp server:
+```
+deb http://ftp.de.debian.org/debian sid main
```

Comment:
Can you remove the region from this URL?

Reply:
Of course!